src/python/dxpy/dx_extract_utils/ExpressionInputsValidator.py

```diff
@@ -0,0 +1,179 @@
+import dxpy
+
+from ..exceptions import (
+    err_exit,
+    PermissionDenied,
+    InvalidInput,
+    InvalidState,
+    ResourceNotFound,
+    default_expected_exceptions,
+)
+from ..utils.resolver import resolve_existing_path, is_hashid, ResolutionError
+
+import json
+import os
+
+
+class ExpressionInputsValidator:
+    """InputsValidator class for extract_assay expresion. Checks for invalid input combinations"""
+
+    def __init__(self, args):
+        self.error_handler = None
```

Comment:
This could possibly have unintended consequences. Best to add `error_handler` explicitly as a parameter to `__init__` so that it can be safely overriden.